easybuild/easyconfigs/c/CDO/CDO-1.9.10-intel-2021b.eb

```
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'CDO'
+version = '1.9.10'
+
+homepage = 'https://code.zmaw.de/projects/cdo'
+description = """CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'intel', 'version': '2021b'}
```

The toolchain can be lowered to `iimpi` as CDO does not use the numeric libraries Intel MKL

```suggestion
toolchain = {'name': 'iimpi', 'version': '2021b'}
```